# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1301591374 25200
#      Thu Mar 31 10:09:34 2011 -0700
# Node ID 4715ce9f911d15c44236fc020086af7a4a4af277
# Parent  88558b8eb8a7955eda7a5aa96661e8c84d3543ab
cor.m: Increase tolerance of %!tests using random data to assure passage.

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,13 @@
+2011-03-31  Rik  <octave@nomad.inbox5.com>
+
+	* statistics/base/cor.m: Increase tolerance of %!tests using random
+	data to guarantee passage.
+
 2011-03-31  Rik  <octave@nomad.inbox5.com>
 
 	* deprecated/module.mk, deprecated/cquad.m: Add deprecated entry for
 	cquad() pointing to quadcc().
 
 2011-03-31  Rik  <octave@nomad.inbox5.com>
 
 	* statistics/base/cor.m: Fix operation with only single input
diff --git a/scripts/statistics/base/cor.m b/scripts/statistics/base/cor.m
--- a/scripts/statistics/base/cor.m
+++ b/scripts/statistics/base/cor.m
@@ -32,15 +32,15 @@ function retval = cor (x, y = x)
   endif
 
   retval = corrcoef (x, y);
 
 endfunction
 
 %!test
 %! x = rand (10, 2);
-%! assert (isequal (cor (x), corrcoef (x)));
+%! assert (cor (x), corrcoef (x), 5*eps);
 %! assert (cor (x(:,1), x(:,2)) == corrcoef (x(:,1), x(:,2)));
 
 %% Test input validation
 %!error corrcoef ();
 %!error corrcoef (1, 2, 3);
 
